client/locations-list-map.js

```diff
@@ -679,20 +681,36 @@ function numberFormat(number, decimalPlaces, decSeparator, thouSeparator) {
 
 /**
  * Adjusts stats in header above map to call out number of markers currently being rendered.
- *
- * @param   $elem   jQuery selector for the stats element
- * @param   count   The number for render
- * @param   total   The total number of entries in the dataset
  */
-function updateStats($elem, count, total) {
-  const prettyMarkerCount = numberFormat(count, 0);
-  const prettyTotalCount = numberFormat(total, 0);
-  $elem.show();
+function updateStats() {
+  // Start with count of location list rows ...
+  let countShown = gLocationsListEntries.length;
+
+  if (showMap) {
+    if (!gMap) {
+      return;
+    }
+
+    const mapBounds = gMap.getBounds();
+    if (!mapBounds) {
+      return;
+    }
+
+    // ... but defer to count of markers in map bounds, when applicable.
+    const countInBounds = (count, marker) => count + mapBounds.contains(marker.getPosition());
+    countShown = gPrimaryMarkers.reduce(countInBounds, 0);
+    countShown += gSecondaryMarkers.reduce(countInBounds, 0);
+  }
+
+  const prettyMarkerCount = numberFormat(countShown, 0);
+  const prettyTotalCount = numberFormat(totalEntries, 0);
+  const stats = $('#list-stats');
```

Comment:
it's convention to prefix jQuery nodes with `$` (i.e. `$stats`)

Reply:
Will do.